src/main/java/org/fundacionjala/coding/franz/FizzBuzz.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.franz;
+
+import java.util.StringJoiner;
+
+/**
+ * this class is FizzBuzz.
+ */
+public class FizzBuzz {
+    private static final int THREE = 3;
+    private static final int FIVE = 5;
+    private static final int ZERO = 0;
+
+    /**
+     * this method run fizzBuzz.
+     *
+     * @param limit is a limit of fizzBuzz
+     * @return string that contains numbers fizzBuzz
+     */
+    public String fizzBuzz(int limit) {
+        StringJoiner joiner = new StringJoiner(" ");
+        for (int i = 1; i <= limit; i++) {
+            joiner.add(isFizzBuzz(i));
+        }
+        return joiner.toString();
+    }
+
+    /**
+     * this method show fizzBuzz.
+     * @param number for check
+     * @return fizz, buzz or fizzbuzz depent of case
+     */
+    public String isFizzBuzz(int number) {
+        if (number % THREE == ZERO && number % FIVE == ZERO) {
+            return "FizzBuzz";
+        }
+        if (number % FIVE == ZERO || String.valueOf(number).toString().contains("5")) {
+            return "Buzz";
+        }
+        if (number % THREE == ZERO || String.valueOf(number).toString().contains("3")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "valueOf" returns a string, there's no need to call "toString()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1858)